### DISCUSSION TITLE
Unable to register as a teacher from UC San Diego

### DISCUSSION THREAD

**Discussion by wasade** [community/community#43749] [GitHub Education]
I am attempting to register as a "teacher" with "education.github.com" and am stuck on the "Upload proof of your academic status" page.

The page seems designed around images of University IDs. However, my University issued ID does not contain a date, which is a required field, so I am unable to submit an image of it.

I am not a student, so I do not have transcripts or other items available in the pulldown menu options of image types. If I select "Other", and submit an image my paystub (cropped) which contains a date and the University name, I am met with an error saying I must submit an image of my University ID. This is unexpected as I am selecting an "Other" image type rather than the ID type.

In short, I cannot use my University ID, and the web form seems to only allow submission of images of IDs. 

Could you please help?

Thank you,
Daniel
